### ISSUE TITLE
PCA/TSNE for embedding visualisation

### ISSUE THREAD

**Issue by alexcoca** [lanpa/tensorboardX#252]
Hey,

I was wondering how to use the SummaryWriter().add_embedding method to visualise neural network representations. I read the docs and saw that all that is necessary to do is to pass a N x D tensor to the function and optionally a list of N labels. Assuming, D>> 3, how can I control which dimensionality reduction method is used to display the plots and its hyperparameters? There seems to be no obvious way to control these through the interface. I was hoping to find out more about this [here](https://github.com/lanpa/tensorboardX/blob/master/demo_embedding.py) but the link to the website appears to be broken. 

Alex

**Comment by lanpa** [comment]
That is moved to: https://github.com/lanpa/tensorboardX/blob/master/examples/demo_embedding.py

You have to control the reduction method on the web interface.
